eureka2-core/src/main/java/com/netflix/eureka2/config/EurekaTransportConfig.java

```diff
@@ -0,0 +1,13 @@
+package com.netflix.eureka2.config;
+
+import com.netflix.eureka2.transport.EurekaTransports;
+
+/**
+ * @author David Liu
+ */
+public interface EurekaTransportConfig {
```

Comment:
There are only two variables right now. We can add a builder if this config grows in the future.